packages/context/src/inject.ts

```diff
@@ -273,6 +354,16 @@ function resolveByTag(
   }
 }
 
+function resolveByTag(
```

Comment:
Wasn't this a part of a different PR - https://github.com/strongloop/loopback-next/pull/857?